thirdparty/grisu2/godot.patch

```diff
@@ -0,0 +1,121 @@
+diff --git a/thirdparty/grisu2/grisu2.h b/thirdparty/grisu2/grisu2.h
```

Comment:
All other patches (except `vhadcd`, which is likely oversight) are in the `patches` subfolder, probably should be the same here.